aws/policy/paas.yaml

```
@@ -24,6 +24,14 @@ Statement:
   - Sid: AllowResourceRestrictedActionsWhichIncurNoFees
     Effect: Allow
     Action:
+      - cloudfront:CreateDistribution
+      - cloudfront:CreateDistributionWithTags
+      - cloudfront:DeleteDistribution
+      - cloudfront:UpdateDistribution
+      - cloudfront:TagResource
+      - cloudfront:UntagResource
+      - cloudfront:ListTagsForResource
```

Anything with a list, get, or describe should be specified as a wildcard.  This helps us stay under the maximum character length per policy that AWS imposes.
For example,
```suggestion
      - cloudfront:List*
```
The wildcard line probably needs to be in the AllowUnrestrictedResourceActionsWhichIncurNoFees block since some of the list actions are not resource restricted.